onyo/cli/rm.py

```diff
@@ -51,18 +51,24 @@
 .. code:: shell
 
     $ onyo rm --message "Bob retired; he won at bingo" admin/Bingo\ Bob/
+
+Remove a building and all of its assets:
+
+.. code:: shell
+
+    $ onyo rm --recursive --message "The big fire..." buildings/20.3/
```

Comment:
Suggestion: "big" -> "great".

It's always a "great fire". ;-P

https://en.wikipedia.org/wiki/Great_Chicago_Fire
https://en.wikipedia.org/wiki/Great_Seattle_Fire
https://en.wikipedia.org/wiki/Great_Fires_of_1871
https://en.wikipedia.org/wiki/Great_Fire_of_London
https://en.wikipedia.org/wiki/Great_fire_of_Hamburg

Reply:
Thanks, after the first 4 sources I was skeptical, but the fifth one really convinced me! ;)